tests/Browser/UserTest.php

```
@@ -30,38 +30,42 @@ public function it_user_can_see_users_on_user_index_and_go_to_the_user_with_link
+        /* Assert */
     }
 
     /**
      * Test i can create a new User.
      */
     #[Test]
-    public function it_i_can_create_a_new_user()
+    public function it_can_create_a_new_user()
     {
         /* Arrange */
         Setting::whereId(1)->update(['max_users' => 10000000]);
```

PHPStan (2.2.5)

[warning] Call to an undefined static method App\Models\Setting::whereId().


(staticMethod.notFound)